test/volume_test.go

```diff
@@ -22,38 +25,61 @@ type VolumeWorkflowSuite struct {
 	volcollService      *service.VolumeCollectionService
 	initiatorGrpService *service.InitiatorGroupService
 }
-
+func (suite *VolumeWorkflowSuite) SetupTest() {
+    fmt.Println("SetupTest")
+	if testResult == "PASS" && result == "PASS" {
+	testResult = "PASS"	
+	} else {
+		testResult = "FAIL"
```

Comment:
Why do we set a test result in the setup function?  Doesn't it belong in a function that runs after the test case is executed?  See https://pkg.go.dev/github.com/stretchr/testify/suite#AfterTest.